.github/workflows/build_and_test.yml

```diff
@@ -26,4 +26,20 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           config: .github/.licenserc.yaml
-
+  test_ci:
+    name: "Test CI"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java-version: [ 11, 17 ]
```

Comment:
We need to support `Java 21` to be consistent with SPARK-43831 .

Reply:
updated workflow for java version - can you please re-approve the workflow ?

Reply:
approved.